index.js

```diff
@@ -1,79 +1,99 @@
 'use strict';
-
-
 const Xray = require('x-ray');
 
+class Probables {
+    static get(dateString, callback) {
 
-const internals = {};
+        let dateComponents = dateString.split('-');
+        if (dateComponents.length !== 3) {
```

Comment:
Write a test to cover this code

Reply:
Nevermind, tests look good.  Not sure what's up with the coverage report.

Reply:
Yeah, I tried to add coverage, and it seems like I did? But still showing as not covered.